src/Maestro/FeedCleaner/.config/settings.Development.json

```diff
@@ -0,0 +1,6 @@
+{
+  "KeyVaultUri": "https://maestroprod.vault.azure.net/",
```

Comment:
Don't we use a local KV for development?

Reply:
Not if Ricardo is testing this against the PROD db!

Reply:
Fixed. Thanks for catching it.